mflowgen/common/rtl/gen_rtl.sh

```diff
@@ -138,7 +138,17 @@ if [ "$use_container" == True ]; then
       trap "docker kill $container_name" EXIT
 
       # aha/bashrc does important things in bg, but does not normally get invoked via 'docker exec'
-      docker exec $container_name /bin/bash -c 'source /aha/aha/bin/docker-bashrc; wait'
+      # docker exec $container_name /bin/bash -c 'source /aha/aha/bin/docker-bashrc; wait'
+
+      docker exec $container_name /bin/bash -c 'source /aha/aha/bin/docker-bashrc' &
+      # Wait 60s for the essentials in bashrc to take effect, then continue
```

Comment:
do you mean 30s?

Reply:
What? 6 x 5 = 60, right? :)

But seriously, though. I took your advice and implemented a '--fast' switch instead, do you want to re-approve?

Also note this will require an aha update, I am filing the PR for that now...